db/schema.rb

```diff
@@ -0,0 +1,53 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema[7.0].define(version: 2023_09_13_194827) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "entities", force: :cascade do |t|
+    t.string "name"
+    t.decimal "amount"
+    t.bigint "user_id", null: false
```

Comment:
- [ ] Great job so far @dieum512 There are a few changes I will suggest you implement and you'll be on your way to the next Milestone :+1: 
- [ ] Just as the previous reviewer mentioned, in the [ERD](https://gitlab.com/microverse/guides/tse/code_review/code_review_guidelines_2_0/-/raw/master/ror/capstone/images/erd_diagram.png) you were given the default `user_id` was replaced by `author_id` as a result, the same thing should have been implemented in your schema but this hasn't been changed so kindly fix this as well :wink:  
- Also, kindly ensure that Rails is aware of the changes when adding references in the entities table :100:

Reply:
Done :ok_hand: